README.md

```diff
@@ -197,14 +197,21 @@ Enter Y after the prompt to read the description, for the purpose of the demo.
 
 It will now start looping through the data elements. If it skips over one it means it was auto-categorised (more on that later).
 
+For this demo, it will only process 20 data elements (out of the 35 total).
+
 ```
-✔ Processing data element 1 of 20
 
-✔ Processing data element 2 of 20
+ℹ 20 left to process in this session
```

Comment:
Is there a way to avoid hard coding :) ? n, n-1, n-2

Reply:
I don't think it is hard coded :)

Reply:
Unless I am not spotting something ...

Reply:
I.e. this is just one example output in the README file